src/json-patch-ot-agent.js

```diff
@@ -39,8 +39,9 @@ JSONPatchOTAgent.prototype.send = function(sequence){
 		path: this.remotePath,
 		value: this.remoteVersion
 	});
-	this.pending.push(sequence);
-	return JSONPatchQueue.prototype.send.call(this, newSequence);
+	var versionedJSONPatch = JSONPatchQueue.prototype.send.call(this, newSequence);
```

Comment:
WDYT about micro-optimization to  `newSequence = JSONPatchQueue.prototype.send.call(this, newSequence);` as we do not have to keep the reference to array on each step?